develop-organize: Update reading section (#37)
open-education-hub/methodology: chapters/develop-organize/reading/reading/README.md

Besides the mentioned guidelines, the linter implements checks for other minor style conventions (such as how to define lists, where should new lines be placed and others).
However, for the majority of cases, the above should be sufficient.

The reading content type is stored as a folder named `reading` in the course repository.

> Each topic should contain a `reading` folder that includes one or multiple Markdown files, depending on how the topic may be broken down in independent reading parts.

Reviewer: Should we give a `tree` example here, it gets a bit confusing?

Author: I thought about this, but ended up concluding that it's not necessary. Now, on a full stomach, it seems indeed that tree example would alleviate any confusion between topic and chapter.

When developing such materials we recommend that:

- the reading material be concise, following the "less is more" principle.
- the first sentence of each section describe what is the intended purpose of the section.
- if the reading material can be expressed in a more visual manner, the latter is preferred.

> - the first person plural is used for explanations: "we derive x", "we download the application" etc.

Reviewer: I would move this up, before the markdown syntax list.

Author: I think it's better this way since all that comes before this list is can be categorized as "objective requirements" (i.e. we can automatically check for them) whereas this list is quite subjective and open to debate.

Reviewer: Fair point, we can leave it like it is now.

Besides the mentioned guidelines, the linter implements checks for other minor style conventions (such as how to define lists, where should new lines be placed and others).
However, for the majority of cases, the above should be sufficient.

> The reading content type is stored as a folder named `reading` in the course repository.
> Each topic should contain a `reading` folder that includes one or multiple Markdown files, depending on how the topic may be broken down in independent reading parts.

Reviewer 2: ```suggestion
The reading content type is stored as a directory named `reading` in the course repository.
Each topic should contain a `reading` directory that includes one or multiple Markdown files, depending on how the topic may be broken down into independent reading parts.
```
Use "directory" instead of "folder" as stated [here](https://discord.com/channels/926757749763756093/1123983780328247326/1141287344301932626).

The below figure highlights the placing of the reading section in a sample repository:

```bash

> |-- chapter1
> |   |-- topic1
>         |-- reading
>             |-- file1.md
>             |-- file2.md
>         |-- media
>         |-- drills
>         |-- guides
>     |-- topic2
>         |-- reading
>         |-- media
>         |-- drills
>         |-- guides
> |-- chapter2
> |-- chapter3

Reviewer 2: ```suggestion
|-- chapter1
|   |-- topic1
        |-- reading
            |-- file1.md
            `-- file2.md
        |-- media
        |-- drills
        `-- guides
    |-- topic2
        |-- reading
        |-- media
        |-- drills
        `-- guides
|-- chapter2
`-- chapter3
```
@razvand used this pattern in the [PR for projects](https://github.com/open-education-hub/methodology/pull/32/files#diff-4c1be4df898340835324c5bcda5429cb7c72517b46ab4e42312de68275600a90R56).

```

When developing such materials we recommend that:

- the reading material be concise, following the "less is more" principle.

> - the first sentence of each section describe what is the intended purpose of the section.

Reviewer 2: ```suggestion
- the first sentence of each section describes what is the intended purpose of the section.
```
